config/locales/en.yml

```diff
@@ -1475,8 +1475,8 @@ en:
       select_certificate_input_label: Select a certificate to issue
     StudentBulkImport__Root:
       csv_data_errors:
-        invalid_affiliation: Affiliation has to be less than 250 characters
         invalid_characters: "%{column_name} column contains one or more invalid characters"
+        invalid_affiliation: Affiliation has to be less than 250 characters
```

Comment:
I think this is a wrong key sort

Reply:
Thanks,🙌

I actually sorted correctly in this https://github.com/pupilfirst/pupilfirst/pull/1478/commits/ec96d9429c09d2b1e6dc98b10a49938957d4ac2f commit but reverted it to easily fix some merge conflicts when merging the master😅

Will sort at the end.